tests/network/salt_master.pm

```diff
@@ -55,6 +76,8 @@ base:
   '*':
     - pkg" > /srv/salt/top.sls));
     assert_script_run("salt '*' state.highstate", 300);
+    assert_script_run "pidstat";
```

Comment:
Why ?

Reply:
Because this package is previously installed by Salt so this is to verify it.

Reply:
ok